Tests/ArrowCsvForeignStorageTest.cpp

```diff
@@ -297,6 +387,26 @@ TEST(DecimalDataTest, DifferentSizesOfDecimal) {
                        {999.9, 9384612.78, 2947583746581.92748}});
 }
 
+TEST(DecimalTest, FragmentedDecimal) {
+  run_ddl_statement("CREATE DATAFRAME decimal_dataframe" + decimal_table_format +
```

Comment:
please move it to TearUp/Down

Reply:
If we move this to SetUp/TearDown exception`Object 'decimal_dataframe' not found" thrown in the test body` will be.

Reply:
why?

Reply:
fixed